opennms-dao/src/test/java/org/opennms/netmgt/dao/support/DefaultResourceDaoTest.java

```diff
@@ -342,6 +347,10 @@ public void testFindNodeResourcesWithResponseTime() throws Exception {
         assertEquals("resource list size", 1, resources.size());
     }
 
+    /**
+     * Rewrite of RmotePoller is still in progress, so ignore this one for now.
+     */
+    @Ignore
```

Comment:
Please add an `TODO cpape` so we can find this later on.

Reply:
done